src/System.Private.ServiceModel/src/System/ServiceModel/Channels/WebSocketTransportSettings.cs

```diff
@@ -21,14 +21,12 @@ public sealed class WebSocketTransportSettings : IEquatable<WebSocketTransportSe
         private WebSocketTransportUsage _transportUsage;
         private TimeSpan _keepAliveInterval;
         private string _subProtocol;
-        private bool _disablePayloadMasking;
 
         public WebSocketTransportSettings()
         {
             _transportUsage = WebSocketDefaults.TransportUsage;
             _keepAliveInterval = WebSocketDefaults.DefaultKeepAliveInterval;
             _subProtocol = WebSocketDefaults.SubProtocol;
-            _disablePayloadMasking = WebSocketDefaults.DisablePayloadMasking;
```

Comment:
We should remove this too then? 

https://github.com/dotnet/wcf/blob/master/src/System.Private.ServiceModel/src/System/ServiceModel/Channels/TransportDefaults.cs#L324